src/js/views/shared/components/list-search/index.js

```diff
@@ -18,6 +17,9 @@ export default Component.extend({
     'watch:change': 'onViewWatchChange',
     'clear': 'onViewClear',
   },
+  initialize({ state }) {
+    if (state.query) this.setState('query', state.query);
```

Comment:
pretty sure this is the default functionality of how a component sets the initial state.  I don't think this `initialize` is needed

Reply:
It doesn't appear to be needed. Can't remember why I added that in the first place tbh...

But just removed it.